config/routes.rb

```diff
@@ -26,6 +26,8 @@
   get 'slack/new' => 'slack#new', as: :new_slack
   get 'slack/auth' => 'slack#update', as: :update_slack
 
+  get '/requests/:id/recap', to: 'requests#recap'
```

Comment:
nice nameing :-D

Reply:
But shouldn't this be vms/requests/:id/recap ?

Reply:
`patch 'requests/request_change_state', to: 'requests#request_change_state', as: 'request_change_state'` should also have a vms/ at the beginning I think (better ask @BjoernDaase )